### ISSUE TITLE
False positives for logging constant keys in another package?

### ISSUE THREAD

**Issue by yizeng** [go-simpler/sloglint#66]
Wondering if I'm missing anything or this is by design: I defined a bunch of logging constant keys in a separate `logging` package and set `no-raw-keys: true`, but I get false positives "raw keys should not be used (sloglint)".

```go
// main.go
package main

import (
	"log/slog"

	"sloglint-example/logging"
)

func main() {
	contentLength := int64(12345)
	url := "https://example.com/file"

	slog.Info("File downloaded successfully",
		slog.Int64(logging.KeyContentLength, contentLength),
		slog.String(logging.KeyURL, url),
	)
}
```

```go
// logging/logging.go
package logging

const (
	KeyContentLength = "content_length"
	KeyURL           = "url"
)

```

```yaml
linters:
  disable-all: true
  enable:
    - sloglint # Enable only sloglint

linters-settings:
  sloglint:
    # Enforce not mixing key-value pairs and attributes.
    no-mixed-args: true
    # Enforce using key-value pairs only (overrides no-mixed-args, incompatible with attr-only).
    kv-only: false
    # Enforce using attributes only (overrides no-mixed-args, incompatible with kv-only).
    attr-only: true
    # Enforce using methods that accept a context.
    context: scope
    # Enforce using static values for log messages.
    static-msg: false
    # Enforce using constants instead of raw keys.
    no-raw-keys: true
    # Enforce a single key naming convention.
    # Values: snake, kebab, camel, pascal
    key-naming-case: snake
    # Enforce putting arguments on separate lines.
    args-on-sep-lines: true
```

Expected: No errors
Actual:
> main.go:13:2: raw keys should not be used (sloglint)
        slog.Info("File downloaded successfully",
        ^

**Comment by tmzane** [comment]
You're right, there was a missing type check for constants in other packages. Fix is on the way :)

**Comment by Jacobbrewer1** [comment]
@tmzane I have a [PR](https://github.com/go-simpler/sloglint/pull/67) with a fix 😄

**Comment by tmzane** [comment]
You beat me to it 😄 I'll take a look shortly, thank you
